webapp/src/pages/success/index.tsx

```diff
@@ -0,0 +1,52 @@
+/** @jsx jsx */
+import { jsx, Button } from 'theme-ui'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import * as styles from './Success.styles'
+import Layout from '../../components/Layout/Layout'
+import HeaderLogo from '../../components/HeaderLogo/HeaderLogo';
+import InfoPage from '../../components/InfoPage/InfoPage';
+
+const Success = () => {
+  const router = useRouter()
+  const { playlist_id: playlistId } = router.query
```

Comment:
It works! Loaded a playlist from my account:

<img width="1680" alt="Screen Shot 2022-02-01 at 5 33 00 PM" src="https://user-images.githubusercontent.com/16858805/152079616-e535cae9-d9ef-4b3a-95d7-4458ab868992.png">